File: server/storage/src/main/java/ru/storage/person/PersonCategory.java
package ru.storage.person;

/**
 * Enum defining the user type folder where user pictures are stored in S3 bucket (e.g. "director", "actor").
 */
public enum PersonCategory {

    DIRECTOR("DIRECTOR"),
    ACTOR("ACTOR"),
    OPERATOR("OPERATOR"),
    PRODUCER("PRODUCER");

    public final String stringValue;

    PersonCategory(String stringValue)  {
        this.stringValue = stringValue;
    }

}

File: web/src/types/PersonCategory.ts
enum PersonCategory {
    DIRECTOR = 'DIRECTOR',
    ACTOR = 'ACTOR',
    PRODUCER = 'PRODUCER',
    OPERATOR = 'OPERATOR',
}

const PersonCategoryLocalized = {
    RU: {
        [PersonCategory.DIRECTOR]: 'Режиссёр',
        [PersonCategory.ACTOR]: 'Актёр',
        [PersonCategory.PRODUCER]: 'Продюсер',
        [PersonCategory.OPERATOR]: 'Оператор',
    },
}

export default PersonCategory
export { PersonCategoryLocalized }
